source/quickstart/k8s.rst

```diff
@@ -6,35 +6,132 @@ Quickstart for Kubernetes
 
 .. default-domain:: minio
 
-This page deploys a MinIO Tenant onto a Kubernetes cluster for use with early
-development and evaluation environments.
+This procedure deploys a basic MinIO server with support for a :ref:`subset of S3 features <minio-installation-comparison>` appropriate for early development and evaluation environments.
```

Comment:
@djwfyi I wanted to try completely simplifying this graf. At this tage even referencing erasure coding might be too much.

The linked comparison chart feels sufficient for users who want to see exactly what this supports.

Reply:
If this feels OK to you I can apply it to the remaining procedures so everything is in sync.

Reply:
If this feels OK to you I can apply it to the remaining procedures so everything is in sync.